src/main/java/org/logstash/dissect/fields/IndirectField.java

```diff
@@ -1,32 +1,38 @@
-package org.logstash.dissect;
+package org.logstash.dissect.fields;
 
 import com.logstash.Event;
+import org.logstash.dissect.ValueResolver;
 
 import java.util.Map;
 
-class IndirectField extends NormalField {
+public final class IndirectField extends AbstractField {
 
     public static Field create(String s) {
         return new IndirectField(s);
     }
 
-    public IndirectField(String s) {
-        super(leftChop(s), 100);
+    private IndirectField(String s) {
+        super(s, 100);
```

Comment:
Is there a reason for the ordinal 100? (also I had to think what this second super class parameter was... . Better use a constant?)